src/components/Booking/View/index.js

```diff
@@ -77,24 +80,31 @@ const BookingView = (props) => {
             </View>
           </View>
 
-          <WarningMessage
-            remainingHolidays={remainingHolidays}
-            booked={booked}
-            potentialHolidays={potentialHolidays}
-            booking={booking}
-          />
+          {rejected
+            ? (
+              <WarningMessage
+                remainingHolidays={remainingHolidays}
+                booked={booked}
+                potentialHolidays={potentialHolidays}
+                booking={booking}
+              />)
+            : null}
+
+          {rejected
+            ? (
+              <View style={styles.buttonContainer}>
+                <RequestButton
+                  updateHoliday={updateHoliday}
+                  submitRequest={submitRequest}
+                  booked={booked}
+                  loading={loading}
+                  remainingHolidays={remainingHolidays}
+                  potentialHolidays={potentialHolidays}
+                  booking={booking}
+                />
+              </View>)
+            : null}
```

Comment:
You could nest the `WarningMessage` and `View` within a `Fragment` and get away with a single `rejected` check, but this works too.

Reply:
Cool, I added the Fragment